app/cypress/e2e/SecvisogramPage/FormEditorTab.cy.js

```diff
@@ -607,4 +607,202 @@ describe('SecvisogramPage / FormEditor Tab', function () {
       '[data-testid="attribute-product_tree-branches-0-branches-0-product-name"] input'
     ).should('have.value', 'Vendor A Product DEF')
   })
-})
+
+  describe('fill functions for revision history', function () {
+    it('fill date of revision', function () {
+      cy.visit('?tab=EDITOR')
+      const now = new Date(2020, 0, 1, 10, 30)
```

Comment:
I'm still wondering - is the `0` correct in the second place?

Reply:
Yes, month indexes start at 0 for the js Date constructor.
See: [Date() constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#monthindex)